test_dir/new_file_1

```diff
@@ -0,0 +1,3 @@
+lets see, here is the file
+well heres what
+what now
```

Comment:
RiflerRick committed on a restricted directory: test_dir/new_file_1in the repository: BB_MCS_Test belonging to Rajdeep Mukherjee(RiflerRick) authored by @RiflerRick in the file test_dir/new_file_1
Owners: no owner file found
 Collaborators: @manasisingh67, @RiflerRick,